### PR TITLE
Added  a note to ignore  DANDI.NO_DANDISET_FOUND  error for final validation before organize

### PR DESCRIPTION
**Pull request by yarikoptic**
We will add --ignore to validate to be used here, but that will be later.

Ref: https://github.com/dandi/dandi-cli/issues/1211  Attn @CodyCBakerPhD

### CODE REVIEW

**Comment by yarikoptic** [comment]
> > We will add --ignore to validate to be used here, but that will be later.
> 
> So the intended call will soon be `dandi validate path/to/nwbfiles --ignore` in order to ignore the 'no dandiset found' error?

`dandi validate path/to/nwbfiles --ignore DANDI.NO_DANDISET_FOUND`

and since @jwodder already has https://github.com/dandi/dandi-cli/pull/1234 , we might indeed not bother with this note, but rather finalize that PR and then I will adjust this PR with `--ignore` invocation and probably adding a note to redo `dandi validate` (without `--ignore`) before `upload` following your (3) recommendation at https://github.com/dandi/dandi-cli/issues/1211#issuecomment-1450308535 .

**Comment by CodyCBakerPhD** [comment]
I do like

```
dandi validate path/to/nwbfiles --ignore DANDI.NO_DANDISET_FOUND
```

since it's nice and explicit about it.

The rest sounds good, thanks for working with me on this

**Comment by yarikoptic** [comment]
ok, fresh dandi-cli was released, please check out rewording here @CodyCBakerPhD

File: docs/13_upload.md [hunk 1]
```diff
@@ -70,20 +70,21 @@ two different servers differ slightly.
...
 
             nwbinspector <source_folder> --config dandi --report-file-path <report_location>.txt
-        
-    4. Once your files are confirmed to adhere to the Best Practices, perform an official validation of the NWB files by running: `dandi validate <source_folder>`. 
-        **If you are having trouble with validation, make sure the conversions were run with the most recent version of `PyNWB` and `MatNWB`.**
-    5. Prepare a dataset folder for upload:
+
+    4. Once your files are confirmed to adhere to the Best Practices, perform an official validation of the NWB files by running: `dandi validate --ignore DANDI.NO_DANDISET_FOUND <source_folder>`.
+        **If you are having trouble with validation, make sure the conversions were run with the most recent version of `dandi`, `PyNWB` and `MatNWB`.**
+    5. Prepare and fully validate now a dataset folder for upload:
```
Comment (CodyCBakerPhD) [inline docs/13_upload.md:76]:
```suggestion
    5. Now, prepare and fully validate again within the dandiset folder used for upload:
```

File: docs/13_upload.md [hunk 2]
```diff
@@ -70,20 +70,21 @@ two different servers differ slightly.
...
             dandi organize <source_folder>
+            dandi validate .
             dandi upload
 
-            For upload to the development server, specify that explicitly via `-i` option, e.g.
-            `dandi upload -i dandi-staging`.
-
+    For upload to the development server, specify that explicitly via `-i` option, e.g.
+    `dandi upload -i dandi-staging`.
+    Also note that validation is also done during `upload`, but ensuring compliance using `validate` prior upload helps to avoid errors during possibly lengthy upload process due to failing validation.
```
Comment (CodyCBakerPhD) [inline docs/13_upload.md:87]: Good explanation 👍

**Comment by yarikoptic** [comment]
Great, thank you @CodyCBakerPhD !  Let's proceed!

**Pull request merged**